titus-api/src/main/java/com/netflix/titus/api/jobmanager/model/job/volume/SaaSVolumeSource.java

```diff
@@ -0,0 +1,66 @@
+package com.netflix.titus.api.jobmanager.model.job.volume;
+
+import java.util.Objects;
+import javax.validation.Valid;
+
+public class SaaSVolumeSource extends VolumeSource {
+
+    @Valid
```

Comment:
I don't think the validation does anything since there isn't a validator to delegate to. My `@NotNull` or `@MinSize` makes more sense?

Reply:
Oh I thought it was just a handy way to make sure it was non-null too? (I thought it was a superset of `@NotNull`)